docs/variants/msi_z690/recovery.md

```diff
@@ -44,3 +44,66 @@ $ flashrom -p linux_spi:dev=/dev/spidev1.0,spispeed=16000 -w 7D25v13/E7D25IMS.13
 ```
 
 * First boot after the recovery process is significantly longer
+
+### SMBIOS unique data recovery
+
+#### Serial number
+
+SMBIOS specification defines two spaces for serial number: the system serial
```

Comment:
`SMBIOS specification` can we turn this into a link?

Reply:
Let's point to specific section we refere to.

Reply:
Added